client/app/components/datagrid/DataGrid.tsx

```diff
@@ -146,10 +146,10 @@ const DataGrid: React.FC<Props> = ({
           "& .MuiDataGrid-columnHeaders": {
             backgroundColor: BC_GOV_BACKGROUND_COLOR_BLUE,
           },
-          "& .MuiDataGrid-columnHeader:first-child": {
+          "& .MuiDataGrid-columnHeader:first-of-type": {
             borderLeft: "none",
           },
-          "& .MuiDataGrid-columnHeader:last-child": {
+          "& .MuiDataGrid-columnHeader:last-of-type": {
```

Comment:
Changed these to clean up warning when running tests: 

`The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".`